checkbox-provider-ce-oem/bin/cpufreq_governors.py

```diff
@@ -719,6 +720,42 @@ def restore_governor(self):
         self.info.set_governor(self.info.original_governor)
 
 
+def probe_governor_module(governor):
+    expected_governor = ["conservative", "powersave", "ondemand",
```

Comment:
I recommend this kind of variable declairation could be put into the `__init__()`. Because it would be easier for some changes in the future.